src/main/kotlin/no/brreg/conceptcatalogue/HarvestEndpoint.kt

```diff
@@ -95,28 +96,28 @@ class HarvestEndpoint(val sqlStore: SqlStore) : CollectionsApi {
         }
 
         definitionBuilder
-            .text(begrep.definisjon?.let { it.toString() } ?: "", "nb")
-            .audience(AudienceType.Audience.Public)
-            .scopeNote(begrep.merknad ?: "", "nb")
-            .scopeBuilder()
+                .text(begrep.definisjon?.let { it.toString() } ?: "", "nb")
```

Comment:
This cannot give valid text/turtlle, because underlying structure is now different.